beetsplug/convert.py

```diff
@@ -142,6 +142,11 @@ def __init__(self):
 
     def commands(self):
         cmd = ui.Subcommand('convert', help=u'convert to external location')
+        cmd.parser.usage = cmd.parser.usage + \
+            u' QUERY\n' + \
+            u'\n' + \
+            u'Convert tracks from your library to external location.\n' + \
+            u'Do not remove those tracks from library.'
```

Comment:
This seems like a good addition, but it may be hard to preserve in the Click transition if we ever do it (c.f. #2190).

I also think the first sentence is a good synopsis but the second sentence might not be the most salient piece of information to include in help output. Can we stick to the first?